test/e2e/framework/create.go

```diff
@@ -655,3 +658,32 @@ func PrettyPrint(item interface{}) string {
 	}
 	return fmt.Sprintf("%+v", item)
 }
+
+type SecretFactory struct{}
```

Comment:
the type and methods would need export comments.

Reply:
or you can just make it private.

Reply:
Making them private sounds much better than adding mostly redundant comments just for the sake of it. Will do.